src/ostree/ot-builtin-refs.c

```diff
@@ -156,13 +176,17 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
   else if (!ostree_repo_list_refs (repo, refspec_prefix, &refs, cancellable, error))
     goto out;
 
-  if (!opt_delete && !opt_create)
+  if (is_list)
```

Comment:
I guess this is more personal preference, though I find the former expression easier to see it's obviously correct just by inspecting the other `else if` and `else` in the tree.